packages/gatsby-theme-docs/src/layouts/internals/sidebar.js

```diff
@@ -234,7 +234,6 @@ const SidebarChapter = (props) => {
       <SpacingsStack scale="s">
         <LinkItem>
           <LinkTitle>{props.chapter.chapterTitle}</LinkTitle>
-          {props.chapter.beta && !props.isGlobalBeta && <BetaFlag />}
```

Comment:
I suppose we don't need the `beta` flag on the chapter level anymore. @nkuehn ?

We should also remove it from the graphql query then.

Reply:
Correct, I forgot we have that feature at all. 
FYI @TimonRey